### PR TITLE
Image Flash Quiz Game

### LINKED ISSUE #2128: [New game]: Image Flash Quiz Game
### 🎮 Game Request

Game logic and basic description
Trivia games are a subcategory of quiz games that feature questions about trivial subjects
It is a game which consists of quizzes in which you have to answer those quizzes about specifically any particular topic/field.

### Point down the features

Game points
1. There will be multiple options for a particular question.
2. There will be a timer in which you have to answer the quiz question.
3. You can see you score on the scoreboard.
4. In order to win, you have to answer max correct questions.

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by shubhisingh184**
forked updation

## PR Description 📜
Added Image Flash Quiz Game

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #2128 

<hr>
 
## Mark the task you have completed ✅

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [X] I have added screenshot for website preview in assets/images 
- [X] I have added entries for my game in main README.md
- [X] I have added README.md in my folder 
- [X] I have added working video of the game in README.md (optional)
- [X] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Image_Flash_Game

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @shubhisingh184 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @shubhisingh184,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — APPROVED** [review]

File: Games/Image_Flash_Game/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Image Flash Game</title>
+
+    <!-- jQuery -->
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
[the remark this answers is not in the document]
Comment (shubhisingh184) [reply Games/Image_Flash_Game/index.html:11]: Thank you for pointing out. I'll  will conduct thorough integrity checks on any scripts loaded from the content delivery network to verify their authenticity and integrity. From my point of view, it's safe. Let me know if you want any changes.

**Comment by kunjgit** [comment]
Hey @shubhisingh184,
 You can't work on a closed issue 👀! 
 Follow the proper guideline and make a new PR again 😀. 
 Happy Hacking 💗

**Comment by kunjgit** [comment]
Thank you  @shubhisingh184 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**